bootstrap/cron.go

```diff
@@ -0,0 +1,50 @@
+package bootstrap
+
+import (
+	"go-google-scraper-challenge/helpers/log"
+	"go-google-scraper-challenge/lib/models"
+	"go-google-scraper-challenge/lib/services/scraper"
+	"time"
+
+	"github.com/go-co-op/gocron"
+)
+
+var scheduler *gocron.Scheduler
+
+func InitCron() {
+	scheduler = gocron.NewScheduler(time.UTC)
+
+	_, err := scheduler.Every(30).Seconds().Do(performScrape)
+	if err != nil {
+		log.Errorln("Fail to setup cron ", err.Error())
+
+		return
+	}
+
+	scheduler.StartAsync()
+}
+
+func performScrape() error {
+	result, err := models.GetOldestPendingResult()
```

Comment:
So with this, it would process one by one keyword, right?

Reply:
Yes, as on the upload API I extract each keyword from the file and save it to the database 1 keyword per one database record